CONTRIBUTING.md

```diff
@@ -1,8 +1,26 @@
 # CLIMADA Contribution Guide
 
-Thank you for contributing to CLIMADA!
+We welcome any contribution to CLIMADA and want to express our thanks to everybody who contributes.
 
-## Overview
+## What Warrants a Contribution?
+
+Anything!
+For orientation, these are some categories of possible contributions we can think of:
+
+* **Technical problems and bugs:** Did you encounter a problem when using CLIMADA? Raise an [issue](https://github.com/CLIMADA-project/climada_python/issues) in our repository, providing a description or ideally a code replicating the error. Did you already find a solution to the problem? Please raise a pull request to help us resolve the issue!
+* **Documentation and Tutorial Updates:** Found a typo in the documentation? Is a tutorial lacking some information you find important? Simply fix a line, or add a paragraph. We are happy to incorporate you additions! Please raise a pull request!
+* **New Modules and Utility Functions:** Did you create a function or an entire module you find useful for your work? Maybe you are not the only one! As a module has to be carefully integrated into the framework, it might help if you talk to us first for designing and planning. You can do that by raising an issue or starting a [discussion](https://github.com/CLIMADA-project/climada_python/discussions) on GitHub.
```

Comment:
```suggestion
* **New Modules and Utility Functions:** Did you create a function or an entire module you find useful for your work? Maybe you are not the only one! For functions, improving things such as plotting or data handling, please feel encouraged tosimply make a direct pull request. Furthermore, as a module has to be carefully integrated into the framework, it might help if you talk to us first about designing and planning. You can do that by raising an issue or starting a [discussion](https://github.com/CLIMADA-project/climada_python/discussions) on GitHub.
```

Reply:
I took up your suggestions and rephrased it again, see https://github.com/CLIMADA-project/climada_python/pull/797/commits/84e248006303d1e749b212c140a7bd71b2177660